src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -77,6 +78,11 @@ def load_arguments(self, _):
                         "--app-insights-key or --app-insights, "
                         "will create a new Application Insights instance in the same resource group.",
                    validator=validate_tracing_parameters_asc_create)
+        c.argument('build_pool_size',
```

Comment:
add preview = true and note this is (enterprise only)

Reply:
OK, added.